tasks/src/tabular-classification/data.ts

```diff
@@ -0,0 +1,54 @@
+import type { TaskData } from "../Types";
+
+import { PIPELINE_DATA } from "../../../js/src/lib/interfaces/Types";
+import { TASKS_MODEL_LIBRARIES } from "../const";
+
+const taskData: TaskData = {
+	datasets: [
+		{
+			description: "Binary classification dataset based on a census on income.",
+			id:          "scikit-learn/adult-census-income",
+		},
+		{
+			description: "Multi-class dataset on iris flower species.",
+			id:          "scikit-learn/iris",
+		},
+	],
+	demo: {
+		inputs: [],
+		outputs: [],
```

Comment:
You can already add an output as done for other classification tasks

Reply:
but I don't know how the dataframe component should look like, since we didn't have it previously. pinging @gary149

Reply:
Alright! I was thinking we could pass just the items of a single row if that helped keep things simpler though. 

E.g. for text classification you could also pass a batch of samples, but the example we show it's just a single example. Also in inference in production, many times the model will just infer from one sample at a time.